test/checkout.int.t.sol

```diff
@@ -1039,13 +946,157 @@ contract Checkout_swapWithoutFeeSpecialOrder is Checkout {
 
     assertEq(
       _bpNori.balanceOf(_namedAccounts.supplier),
-      (certificateAmount * customPriceMultiple) / 100 / 2 // divide to account for price multiple scale and then holdback percentage of 50%
+      (certificateAmount * customPriceMultiple) / 100 / 2 // divide to account for price multiple scale and holdback percentage of 50%
     );
     assertEq(_bpNori.balanceOf(_namedAccounts.feeWallet), 0);
   }
 }
 
-contract Checkout_swapFromSupplierWithoutFeeSpecialOrder is Checkout {
+contract Checkout_swapWithoutFeeSpecialOrder_specificVintages is Checkout {
+  uint256 ownerPrivateKey = 0xA11CE;
+  address owner = vm.addr(ownerPrivateKey);
```

Comment:
Use named accounts

Reply:
makes sense -- this might have been an artifact of adapting permit-based tests which need access to the private key for signing